share/jupyterhub/templates/spawn_pending.html

```diff
@@ -34,10 +39,54 @@
   {{ super() }}
   <script type="text/javascript">
     require(["jquery"], function($) {
+      var base_url = window.jhdata ? window.jhdata.base_url : "{{ base_url }}";
+      var user = window.jhdata ? window.jhdata.user : "{{ user.name }}";
+
       $("#refresh").click(function() {
         window.location.reload();
       });
 
+      // Try to load JHAPI for stop server functionality
+      require(["jhapi"], function(JHAPI) {
```

Comment:
Why was this moved down? Leaving it as the top-level require made more sense to me and was simpler. Let's put that back.